.github/workflows/python-package.yml

```diff
@@ -0,0 +1,56 @@
+name: Python package
+
+on: [push]
+
+env:
+  LINUX_BASEKIT_URL: https://registrationcenter-download.intel.com/akdlm/irc_nas/17769/l_BaseKit_p_2021.2.0.2883_offline.sh
+  LINUX_DPCPP_COMPONENTS_WEB: intel.oneapi.lin.dpcpp-cpp-compiler
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.7, 3.8, 3.9]
+
+    env:
+      ONEAPI_ROOT: /opt/intel/oneapi
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
```

Comment:
It fetches whole repository history. Is not necessary but with it we can see package version based on tag.